tests/model/test_model.py

```diff
@@ -1528,6 +1528,48 @@ def test_fetch_raw_message_content(
         assert self.display_error_if_present.called == display_error_called
         assert return_value == expected_raw_content
 
+    @pytest.mark.parametrize(
+        "response, expected_read_receipts, display_error_called, feature_level",
+        [
+            case(
+                {"msg": "", "result": "success", "user_ids": [3, 7, 9]},
+                [3, 7, 9],
+                False,
+                139,
```

Comment:
Sorry, wrote 139 as the cusp version because read it in the original issue which I based this PR on. Changing the text in commit and PR to 137.

The test case is meant for 139 though, that was intentional.